src/styles/components/loginPage.scss

```diff
@@ -0,0 +1,62 @@
+@import '../styles/_settings';
+
+.login-container {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  padding: 4rem;
+
+  .form-container {
```

Comment:
This should go all in one line. Please check the css that you modified for any other cases like this.